src/ari_sxn_simbeamline/caproto_servers/baffle_slit.py

```diff
@@ -24,10 +24,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)  # call the PVGroup __init__ function
 
     # Add the baffle motor PVs.
-    top = SubGroup(FakeMotor, velocity=0.1, precision=3, user_limits=(-1, 20), prefix=':Top')
-    bottom = SubGroup(FakeMotor, velocity=0.1, precision=3, user_limits=(-20, 1), prefix=':Bottom')
-    inboard = SubGroup(FakeMotor, velocity=0.1, precision=3, user_limits=(-20, 1), prefix=':Inboard')
-    outboard = SubGroup(FakeMotor, velocity=0.1, precision=3, user_limits=(-1, 20), prefix=':Outboard')
+    top = SubGroup(FakeMotor, velocity=1, precision=3, user_limits=(-13, 40), prefix=':Top')
```

Comment:
Not your mistake Jiemin but can I ask you to make sure all of these lines are less than 80 characters long?